### PR TITLE
Remove levelup dependency

### LINKED ISSUE #114: security Issue with bl
Hi, the new version npm tell me there is a security issue : 

                                                                                
                       === npm audit security report ===                        
                                                                                
                                                                                
                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           
                                                                                
                                                                                
  Moderate        Memory Exposure                                               
                                                                                
  Package         bl                                                            
                                                                                
  Patched in      >=0.9.5 <1.0.0 || >=1.0.1                                     
                                                                                
  Dependency of   level-sublevel                                                
                                                                                
  Path            level-sublevel > levelup > bl                                 
                                                                                
  More info       https://nodesecurity.io/advisories/596                        
                                                                                

[!] 1 vulnerability found - Packages audited: 498 (0 dev, 0 optional)
    Severity: 1 Moderate

### PR DESCRIPTION
**Pull request by eight04**
Fixes #114.
Fixes #101.

I have to say that I have no idea how `levelup` works. I took some code from #114 and implemented a readable stream that maps `level-iterator-stream` to old API.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,15 +23,14 @@
...
-    "pull-level": "~1.1.1",
     "rimraf": "~2.1.4",
     "shasum": "0.0.2",
     "stream-to-pull-stream": "~1.2.0",
-    "tape": "~2.14.0",
+    "tape": "^4.9.1",
     "through": "~2.3.4"
   },
   "scripts": {
-    "test": "set -e; for t in test/*.js; do node $t; done"
```
[the remark this answers is not in the document]
Comment (eight04) [reply package.json]: I use Windows.

**Comment by eight04** [comment]
> okay, if this works, can I add you as a maintainer?
> I don't use this module anymore myself. (I use flumedb now)
> I will add you as collaborator and as npm owner so you can publish new versions

Sure. But I don't use this module either. This PR is just to fix the security warning in dependencies.

BTW, I saw [this thread](https://github.com/Level/community/issues/14). Should README be updated to suggest using `subleveldown`?

**Comment by dominictarr** [comment]
okay then... merged into 6.6.3

**Comment by vweevers** [comment]
This should not have been a patch release. It breaks read streams altogether and introduces dependencies like `level-codec` that have dropped node versions (which warrants a major) and very likely contain more subtle changes that are incompatible with `level-sublevel`.

Unless someone is willing to invest time in `level-sublevel`, I think the best course of action is to revert these changes and release as 6.6.4. And perhaps `npm deprecate` it and point people to `subleveldown` - which @ralphtheninja and I actively maintain.

**Comment by mhart** [comment]
Yeah, will lock down the version in dynalite and look at moving to subleveldown

**Comment by dominictarr** [comment]
reverted this in 6.6.5, sorry everyone

**Pull request merged**